training/ilab-wrapper/ilab

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
 # Template values replaced by container build
+ENDPOINT_URL="__REPLACE_ENDPOINT_URL__"
+DATE="__REPLACE_DATE__"
+TAXONOMY_PATH="__REPLACE_TAXONOMY_PATH__"
```

Comment:
I think we should put these changes in another name ilab-full for example, just so that everyone still functions until we finish

Reply:
sorry to be clear i mean instead of against ilab another file, that way the image is still consumable then once done we can flip it